src/backend/vulkan/src/lib.rs

```diff
@@ -109,7 +114,7 @@ lazy_static! {
         );
 }
 
-pub struct RawInstance(ash::Instance, Option<DebugMessenger>);
+pub struct RawInstance(ash::Instance, Option<DebugMessenger>, Option<vk::KhrGetPhysicalDeviceProperties2Fn>);
```

Comment:
at this point we should name the fields (can follow up, not strictly required her)